<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProcessDialog</class>
 <widget class="QDialog" name="ProcessDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>541</width>
    <height>589</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>STeAMM - Get MODIS Swaths</string>
  </property>
  <widget class="QTextBrowser" name="text_StatusConsole">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>400</y>
     <width>511</width>
     <height>171</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_StatusConsole">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>380</y>
     <width>106</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Status Messages:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_DataDir">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>278</y>
     <width>411</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="button_BrowseDataDir">
   <property name="geometry">
    <rect>
     <x>435</x>
     <y>275</y>
     <width>93</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Browse...</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_ProductID">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>25</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>MODIS Data Product ID:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="cbo_ProductID">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>25</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_SwathID">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>65</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>MODIS Swath ID:</string>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_ProcessYear">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>105</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Processing year:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="cbo_SwathID">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>65</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="cbo_ProcessYear">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>105</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_DoYStart">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>175</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_JulianDaysDesc">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>301</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Select the beginning and end dates, in julian days:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_HDFConvert">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Convert HDF files to Geotiff</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_DoYStart">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>175</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Beginning of year:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_DoYEnd">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>175</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>End of year:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_DoYEnd">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>175</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>366</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_DataDir">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>253</y>
     <width>206</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Download to this directory file path:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="Line" name="lineDivider">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>360</y>
     <width>511</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_Credentials">
   <property name="geometry">
    <rect>
     <x>305</x>
     <y>20</y>
     <width>221</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>EarthData Login</string>
   </property>
   <widget class="QLabel" name="label_Username">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>35</y>
      <width>66</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Username:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Password">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>70</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Password:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Username">
    <property name="geometry">
     <rect>
      <x>85</x>
      <y>35</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Password">
    <property name="geometry">
     <rect>
      <x>85</x>
      <y>70</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="button_Start">
   <property name="geometry">
    <rect>
     <x>325</x>
     <y>330</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="button_Reset">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>330</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>button_Reset</sender>
   <signal>pressed()</signal>
   <receiver>ProcessDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>464</x>
     <y>341</y>
    </hint>
    <hint type="destinationlabel">
     <x>257</x>
     <y>315</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_Start</sender>
   <signal>pressed()</signal>
   <receiver>ProcessDialog</receiver>
   <slot>exec()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>353</x>
     <y>350</y>
    </hint>
    <hint type="destinationlabel">
     <x>147</x>
     <y>324</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
